restonomer-core/src/main/scala/com/clairvoyant/restonomer/core/transformation/RestonomerTransformation.scala

```diff
@@ -82,3 +82,13 @@ case class ReplaceStringInColumnValue(
     restonomerResponseDF.replaceStringInColumnValue(columnName, pattern, replacement)
 
 }
+
+case class AddSuffixToColumnNames(
+    suffix: String,
+    columnNames: List[String] = null
```

Comment:
Initialise it as empty list and not null

Reply:
Done!